model/Core/Datatypes/DateTime.md

```diff
@@ -15,6 +15,7 @@ The specific format is one of the most commonly used ISO-8601 formats.
 ## Metadata
 
 - name: DateTime
+- SubclassOf: xsd:string
```

Comment:
These are not needed, since we've said that all Datatypes are strings with pattern restrictions.

Reply:
SpdxId is a string without pattern restrictions but with the xsd:anyURL restriction.  Other Datatypes could be strings with length restrictions.  Both the base class (to allow the RDF-defined xsd datatypes) and other datatype keywords (such as length) are needed.

Reply:
I repeat, all our Datatypes are strings with pattern restrictions. It makes no sense to plan for something we do not have any use for.

Reply:
It is not a plan. We already have PositiveIntegerRange and could well need NonNegativeIntegerRange if Snippet byte ranges are to be zero-based.  We already have Package downloadlocation and Package concludedLicense, which are semantically different types.

You see no use for these, "we" (other members of the tech team) do see a use for them.  So again, is there any harm?